x-pack/plugins/cases/public/components/user_actions/comment/comment.test.tsx

```diff
@@ -263,6 +265,33 @@ describe('createCommentUserActionBuilder', () => {
       expect(result.getByTestId('comment-externalReference-.test')).toBeInTheDocument();
       expect(screen.getByText('Attachment actions')).toBeInTheDocument();
     });
+
+    it('renders the delete attachment action correctly', async () => {
```

Comment:
Should we test that clicking the delete icon performs an action? Calls a callback or something?